integration-tests/tests/workspace-dashboard.js

```diff
@@ -173,6 +192,10 @@ const setAzureAjaxMockValues = async (testPage, namespace, name, workspaceDescri
         filter: { url: detailsUrl },
         fn: () => () => Promise.resolve(new Response(JSON.stringify(azureWorkspaceDetailsResult), { status: 200 }))
       },
+      {
+        filter: { url: workspaceResourcesUrl },
+        fn: () => () => Promise.resolve(new Response(JSON.stringify(azureWorkspaceResourcesResult), { status: 200 }))
+      },
```

Comment:
I like that we're trying out this mocking strategy. I've always worried about drift between the mock and real APIs, and this is a chance to see whether those worries have merit. I kind of doubt we'll see them change very often.